kadmin/kadmin.go

```diff
@@ -25,6 +26,48 @@ type Kadmin interface {
 	ConfigUpdater
 	TopicConfigLister
 	SraSetter
+	ClusterConfigLister
+	BrokerConfigLister
+}
+
+type ClusterConfigLister interface {
+	GetClusterConfig() (ClusterConfig, error)
+}
+
+type BrokerConfigLister interface {
+	GetBrokerConfig(brokerID int32) (BrokerConfig, error)
```

Comment:
This should only return `ClusterConfigListingStartedMsg`

Reply:
Why not a `tea.Msg` type here?

Reply:
Yes this should indeed by a `tea.Msg` in the interface but the impl. should effectively return something like `ClusterConfigListingStartedMsg`